tyr/policies/ec2.py

```diff
@@ -242,8 +276,10 @@
         "sns:Subscribe"
       ],
       "Resource": [
-        "arn:aws:sns:us-east-1:761584570493:s-outpost",
-        "arn:aws:sns:us-east-1:761584570493:alyx3"
+        "arn:aws:sns:us-east-1:761584570493:stage-outpost",
```

Comment:
Thanks, I have removed these now.  To be honest I was just worried there might be some using an old name - I should have just checked really.